python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -41,8 +41,14 @@ def run_kuberay_autoscaler(cluster_name: str, cluster_namespace: str):
             print(f"Will check again in {BACKOFF_S} seconds.")
             time.sleep(BACKOFF_S)
 
-    # The Ray head container sets up the log directory. Thus, we set up logging
-    # only after the Ray head is ready.
+    # For kuberay's autoscaler integration, the autoscaler runs in a sidecar container
+    # in the same pod as the main Ray container, which runs the rest of the Ray
+    # processes.
+    # The logging configuration here is for the sidecar container, but we need the
+    # logs to go to the same place as the head node logs because the autoscaler
+    # is allowed to communicate with another Ray process (the log monitor) via logs
+    # in that directory. However, the Ray head container sets up the log directory.
```

Comment:
Could you split this sentence up?

Reply:
Definitely. I've split it up and reworded it - what do you think now?

Reply:
Looks good, thanks!